samples/snippets/src/main/java/pubsub/RemoveDeadLetterPolicyExample.java

```diff
@@ -61,9 +58,7 @@ public static void removeDeadLetterPolicyExample(
       // Construct a field mask to indicate which field to update in the subscription.
       FieldMask updateMask =
           FieldMask.newBuilder()
-              .addPaths("dead_letter_policy.dead_letter_topic")
-              // A default of 5 is applied upon successful update.
-              .addPaths("dead_letter_policy.max_delivery_attempts")
+              .addPaths("dead_letter_policy")
```

Comment:
This, like it's predecessor looks wrong - shouldn't this be a constant defined in the library?

Reply:
Delivery attempt is not a constant defined in the library. In fact, it is best-effort only, and when no dead letter policy is attached, users won't be able to access delivery attempts.

Reply:
Understood - if it's understood as something important and understood as something special - it should be defined in the library in some way.